lib/static/new-ui/experiments/time-travel/components/SnapshotsPlayer/utils.ts

```diff
@@ -82,6 +69,66 @@ export const loadSnapshotsFromZip = async (zipUrl: string): Promise<NumberedSnap
     });
 };
 
+interface LoadSnapshotsFromZipOptions {
+    abortSignal?: AbortSignal;
+    onDownloadProgress?: (progressPercent: number) => void;
+}
+
+export const loadSnapshotsFromZip = async (zipUrl: string, {abortSignal, onDownloadProgress}: LoadSnapshotsFromZipOptions): Promise<NumberedSnapshot[]> => {
+    const response = await fetch(zipUrl, {signal: abortSignal});
+    if (!response.ok) {
+        throw new Error([
```

Comment:
will the report crash with this error?

Reply:
No, I've tested that. A warning would be printed to console.